firebase/test/core/data/repositories/memos-repository.spec.ts

```diff
@@ -0,0 +1,136 @@
+import * as assert from "assert";
+import * as sinon from "sinon";
+import { FirestoreGateway } from "#data/gateways/firestore-gateway";
+import { MemosRepository } from "#data/repositories/memos-repository";
+import Ajv2020 from "ajv/dist/2020";
+import { SchemaValidator } from "#data/schemas/schema-validator";
+import { Memo } from "#domain/models/memo";
+import SerializationError from "#faults/errors/serialization-error";
+import createSinonStub from "#test/sinon-stub";
+
+describe("MemosRepository", () => {
+  const firestoreStub = createSinonStub(FirestoreGateway);
+  const schemaValidator = new SchemaValidator(new Ajv2020());
```

Comment:
> This is not mocked/stubbed, we shouldn't rely on anything other than the unit being tested

Again, not sure why, but why isn't the schema being stubbed as well, if we are testing the repository?

Reply:
You're right. Also added some asserts that the validation is being made when setting and getting the memos